keyboards/planck/keymaps/pascamel/config.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "config_common.h"
```

Comment:
This line should be removed, since it's not needed, actually.

Reply:
@drashna just to clarify, are you referring to line 1 or 3?

Reply:
Line 3.   This is included in the main keyboard's config.h, so it's redundant.

Reply:
Gotcha! will update soon.